pinot-filesystem/pom.xml

```diff
@@ -49,5 +49,13 @@
       <artifactId>commons-configuration</artifactId>
       <version>1.6</version>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
```

Comment:
Is this being used? Didn't find any references.

Reply:
I'm using it in PinotFSFactory